pkgs/development/python-modules/dwdwfsapi/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, requests
+, ciso8601
+}:
+
+buildPythonPackage rec {
+  pname = "dwdwfsapi";
+  version = "1.0.3";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-PX1b1msaZH8HKVBo3GU7TOr8Lo7INLjjJBkDHHs6mzk=";
+  };
+
+  propagatedBuildInputs = [ requests ciso8601 ];
```

Comment:
One dependency per line, to improve diffs and merges.

Reply:
Also you are missing `urllib3`, which gets propagated by `requests`, so this doesn't cause any error right now. And since they're not importing `urllib3` directly, I think this is fine.

Reply:
Updated, thanks.